src/themes/types.ts

```diff
@@ -99,12 +99,10 @@ export type ColorPalette = ExtendablePalette<
 /**
  * A type for the generic color scheme of a component based on CSS property names
  */
-export type ColorScheme = {
-  foreground: string
-  background: string
-  border: string
-  shadow: string
-}
+export type ColorScheme = Record<
+  'foreground' | 'background' | 'border' | 'shadow' | 'lighterBackground',
```

Comment:
What is `lighterBackground` representing here? Maybe if we need different background it is result of some state inside the component. Should we consider adding object containing colors for the states: hovered, focus etc?

Reply:
Or maybe it is a color of other type of the Menus?

Reply:
not relevant anymore, see #756